frontend/src/teams/team/team-route.tsx

```diff
@@ -54,7 +62,7 @@ const TeamRoute = () => {
           <TeamForm onFormSubmit={onFormSubmit} team={team} />
         </ContentCard>
         <ContentCard fullWidth={true}>
-          <PlayersList team={team} onFormSubmit={onFormSubmit} />
+          <PlayersList team={team} deletePlayer={deletePlayer} />
```

Comment:
Why are you passing whole `team` object? This is `PlayersList` component. Pass `players` and `teamId`. In this example it is maybe not that important but what when you have object with 10 fileds? it is meaningless to pass whole object, and it builds dependency, making component much harder to reuse.